src/test/java/teammates/test/cases/browsertests/ScrollUnitTest.java

```diff
@@ -0,0 +1,95 @@
+package teammates.test.cases.browsertests;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import teammates.common.util.AppUrl;
+import teammates.common.util.Const;
+import teammates.common.util.ThreadHelper;
+import teammates.test.pageobjects.InstructorFeedbackEditPage;
+
+/**
+ * Unit Test for functions in scrollTo.js.
+ * All tests are done on InstructorFeedbackEditPage page to get direct access to JS functions
+ */
+public class ScrollUnitTest extends BaseUiTestCase {
+
+    private static final String SCROLL_TO_POSITION_SCRIPT = "if (arguments[1] === undefined || arguments[1] === null) {\n"
+            + "        $(window).scrollTop(arguments[0]);\n"
+            + "    } else {\n"
+            + "        $('html, body').animate({ scrollTop: arguments[0] }, arguments[1]);\n"
+            + "    }";
+
+    private static final String VERTICAL_SCROLL_VALUE = "return window.scrollY";
+
+    private InstructorFeedbackEditPage feedbackEditPage;
+
+    private String instructorId;
+    private String courseId;
+    private String feedbackSessionName;
+
+    @Override
+    protected void prepareTestData() {
+        // No need  of data. Dummy data only for scrolling purposes
+        testData = loadDataBundle("/InstructorFeedbackEditPageUiTest.json");
+        removeAndRestoreDataBundle(testData);
+
+        instructorId = "CFeedbackEditUiT.instructor";
+        courseId = "CFeedbackEditUiT.CS2104";
+        feedbackSessionName = "First Session";
+    }
+
+    @BeforeClass
+    public void classSetup() {
+        feedbackEditPage = getFeedbackEditPage();
+    }
+
+    @Test
+    public void allTests() {
```

Comment:
I'm thinking whether we should continue to follow this style of testing by nesting all the tests in an `allTests()` function. My experience is that with this, all new tests that is created will be included into this function. 

My understanding is that having `@Test` does not ensure that test runs in specific sequences :https://stackoverflow.com/questions/2669576/order-of-execution-of-tests-in-testng but in this case I think order is not important so we probably should just use the default annotations of `@Test` ? 

Even if order is important in some tests, we should attempt to decouple them with better mocking.